todo_app/templates/index.html

```diff
@@ -7,11 +7,37 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <form name="Add Todo" method = "POST" action="add">
+    <label for="taskname">Task Title : </label><br>
+    <input type="text" id="taskname" name="taskname"><br>
+    <label for="taskdesc">Task description : </label><br>
+    <input type="text" id="taskdesc" name="taskdesc"><br>
+    <label for="status">Task Status : </label><br>
+    <select name="status" id="status">
+      {%for status in statuses%}
+      <option value={{ status['id'] }}>{{ status['name'] }}</option>
+      {%endfor%}
+      </select><br>
+    <input type="submit" value = "Save">
+  </form>
+
+
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        <table>
+          <th>ID</th><th>Title</th><th>Status</th>
+          {%for item in items%}
+          <tr>
+            <td>{{item.id}}</td>
+            <td>{{item.title}}</td>
+            <td>{{item.status}}</td>
+          </tr>
+          {%endfor%}
+        </table>
+        <a href = "/add"> Add a task</a>
```

Comment:
This link seems to be a plan for an alternative approach for adding tasks that hasn't been implemented, so in true YAGNI fashion ("you ain't gonna need it") we should get rid of this!